src/test/suite/fixtures/recorded/languages/python/chuckEveryArgRed.yml

```
@@ -0,0 +1,27 @@
+languageId: javascript
```

This should probably be 
```suggestion
languageId: python
```

---

Yeah that's definitely not right

---

I re recorded these tests.